src/main/scala/uk/gov/hmrc/initprototype/HerokuGenerateSpinDownListTask.scala

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.initprototype
+
+import os.Path
+import play.api.libs.json._
+import play.api.libs.json.Reads._
+import play.api.libs.functional.syntax._
+
+import java.time.Instant
+
+class HerokuGenerateSpinDownListTask(config: HerokuConfiguration) {
+
+  case class SpinDownCandidate(name: String, lastUpdated: Instant)
+  implicit val spinDownCandidate: Reads[SpinDownCandidate] = (
+    (JsPath \ "name").read[String] and
+      (JsPath \ "updated_at").read[Instant]
+  )(SpinDownCandidate.apply _)
+
+  def generateSpinDownList(outputPath: Path): Unit = {
+    val newCandidates = herokuGet(s"${config.baseUrl}/teams/digital-hmrc-gov/apps")
+      .as[List[SpinDownCandidate]]
+      .filter(candidate =>
+        candidate.lastUpdated.isBefore(config.dateWhereAppsConsideredStillActive)
```

Comment:
I would be very interested in suggestions for better name for this config entry, I could maybe make it `Period.ofDays(84)` and use it like this which might be more readable

```
candidate.lastUpdated.isBefore(Instant.now.minus(config.periodOfTimeBeforeAppConsideredInactive))
```

Or could rename it to "dateWhereAppsConsideredInactive", or maybe it's ok 🤷‍♂️

Reply:
Everything looks good to me, I do like your suggestion of making it Period.ofDays(84) for readability sakes.